controller/gatewayclass/controller_reconciler_utils.go

```diff
@@ -0,0 +1,65 @@
+package gatewayclass
+
+import (
+	"context"
+	"strings"
+
+	k8serrors "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	gatewayv1 "sigs.k8s.io/gateway-api/apis/v1"
+
+	operatorv1beta1 "github.com/kong/gateway-operator/api/v1beta1"
+	"github.com/kong/gateway-operator/pkg/consts"
+	k8sutils "github.com/kong/gateway-operator/pkg/utils/kubernetes"
+)
+
+// getAcceptedCondition returns the accepted condition for the GatewayClass, with
+// the proper status, reason and message.
+func getAcceptedCondition(ctx context.Context, cl client.Client, gwc *gatewayv1.GatewayClass) (*metav1.Condition, error) {
+	reason := string(gatewayv1.GatewayClassReasonAccepted)
+	message := []string{}
```

Comment:
```suggestion
	messages := []string{}
```
Since it's a list, I suggest usage of plural